src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -483,4 +485,59 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
+    /**
+     * Removes templates and actual values filled in place of templates
+     * for status messages
+     * 
+     * Example : "You are enrolled in %s" and "You are enrolled in CS2103"
+     *            are converted to "You are enrolled in "
+     *            
+     * @param statusMessage
+     * @return status message without variables and actual values
+     */
+    public static String removeVariables(String statusMessage) {
```

Comment:
Ok. I feel this is very fragile too. Any change to `Const.java` status messages may affect this and cause it to fail.